doc/man1/openssl.pod

```diff
@@ -763,12 +763,6 @@ OpenSSL command to generate an alternative chain.
 
 =over 4
 
-=item B<-xchain_build>
```

Comment:
I did not notice this one before - this deletion does not seem right in this PR.

Reply:
It must have been a rebase mistake.  Restored the text.